tests/test_nodes.py

```diff
@@ -600,8 +600,12 @@ def test_dereference_missing() -> None:
 )
 def test_validate_and_convert_none(make_func: Any) -> None:
     node = make_func("???", is_optional=False)
+    ref_type_str = node._metadata.ref_type.__name__
```

Comment:
use the same function: type_str.

Reply:
Done in https://github.com/omry/omegaconf/pull/754/commits/9058f9df3d2996963b35eddc133b7a229e2e27d6.